public/index.js

```diff
@@ -48,4 +51,13 @@ async function createWebSocket (loc) {
   })
 }
 
+// Debug logging can be enabled by entering logDebug(true) in the console.
+// Your setting will persist across sessions.
+window.loggingDebug = window.localStorage.getItem('loggingDebug') === '1'
+window.logDebug = yes => {
+  window.loggingDebug = yes
+  window.localStorage.setItem('loggingDebug', yes ? '1' : '0')
+  return 'debug logging set to ' + (yes ? 'true' : 'false')
+}
+
```

Comment:
I wanted to reduce the noise in the console so that I could see some temporary prints I was putting in. Turning off "debug logging" disables the websocket messages and pings and new blocks. This verbose output is off by default, though, which has potential consequences. For example, if you see something wrong, it's not always true anymore that you can open your console and see the block history. The flip side is that errors will be obvious, because they won't get buried over time.